administrator/language/en-GB/plg_editors_codemirror.ini

```
@@ -3,42 +3,49 @@
 ; License GNU General Public License version 2 or later; see LICENSE.txt
 ; Note : All ini files need to be saved as UTF-8
 
-PLG_CODEMIRROR_FIELD_ACTIVELINE_COLOR_LABEL="Active Line Colour"
-PLG_CODEMIRROR_FIELD_ACTIVELINE_LABEL="Highlight Active Line"
 PLG_CODEMIRROR_FIELD_AUTOCLOSEBRACKET_LABEL="Bracket Completion"
-PLG_CODEMIRROR_FIELD_AUTOCLOSETAGS_LABEL="Tag Completion"
 PLG_CODEMIRROR_FIELD_CODEFOLDING_LABEL="Code Folding"
-PLG_CODEMIRROR_FIELD_FONT_FAMILY_LABEL="Font"
-PLG_CODEMIRROR_FIELD_FONT_SIZE_LABEL="Font Size (px)"
+PLG_CODEMIRROR_FIELD_CUSTOM_EXTENSIONS_LABEL="Custom Extensions"
+PLG_CODEMIRROR_FIELD_CUSTOM_EXTENSIONS_METHOD_DESCR="Method name, provided by module for extension initialisation. Or multiple, separated by comma. Eg: bracketMatching (from module @codemirror/language)."
```

```suggestion
PLG_CODEMIRROR_FIELD_CUSTOM_EXTENSIONS_METHOD_DESCR="Method name, provided by module for extension initialisation. Or multiple, separated by comma. Eg: bracketMatching (from module @codemirror/matchbrackets)."
```

I knbow its only an example but it makes sense for it to be correct ;)

---

sorry my mistake